recipes/rauth/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: rauth
+  version: "0.7.3"
+
+source:
+  fn: rauth-0.7.3.tar.gz
+  url: https://pypi.python.org/packages/43/aa/7c8e852275394d65ac5bf3ac9945ecaafe4d083089e09cb0a267efea389a/rauth-0.7.3.tar.gz
```

Comment:
Please use the canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18